### PR TITLE
Update required CentOS version

### LINKED ISSUE #4674: Update required CentOS version for UCP 2.2
In [Deploy Enterprise Edition on Linux servers](https://docs.docker.com/datacenter/install/linux/), Engine 17.06 requires CentOS 7.4.

### PR DESCRIPTION
**Pull request by JimGalasyn**
Fixes #4674 .

### CODE REVIEW

**Comment by mstanleyjones** [comment]
Oh well, guess I was too late.

**Comment by JimGalasyn** [comment]
I saw your review, just as I hit the Merge button. >< Got another PR posted now.

File: datacenter/install/linux.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ EE have a minimum of:
 
 Also, make sure the hosts are running one of these operating systems:
 
-* CentOS 7.1 or 7.2
+* CentOS 7.4
```
Comment (thaJeztah) [inline datacenter/install/linux.md]: IIRC, CentOS itself even refers to this as `CentOS 7` (as it is a rolling release model, so basically there is no `.1`, `.2`, `.3`)

Comment (JimGalasyn) [reply datacenter/install/linux.md]: For sure. @thaJeztah, check out this PR: #4678.

**Pull request merged**